.github/pull_request_template.md

```diff
@@ -1,17 +1,31 @@
-This PR CHANGES.
+<!---
+Briefly describe the change.
+-->
+
+This PR ...
 
 ## Todos
 
-- [ ] Todo
+<!---
+Consider creating a TODO list to help others understand the progress of work in a WIP pull request.
+-->
 
-## Screenshots
+- [ ] TODO
 
-- Screenshot
+## Screenshots
 
----
+<!---
+Use the GitHub drag&drop feature to upload default-sized Daedalus window screenshots
```

Comment:
I think should be worth mentioning that screenshots cannot have drop shadow. On OSX this can be achieved by pressing `Command + Shift + 4 + Space bar` 

![Screen Shot 2021-12-24 at 09 36 30](https://user-images.githubusercontent.com/2846918/149132627-26973486-f537-41f8-ae66-841765d3040f.png)